src/Sylius/Component/Association/Model/Association.php

```diff
@@ -39,7 +38,7 @@ class Association implements AssociationInterface
     protected $owner;
 
     /**
-     * @var Collection<AssociableInterface>
+     * @var AssociableInterface[]
```

Comment:
I'm not sure, as it implied that there can be collection of ``AssociableInterface`` objects **or** some